sentinelsat/sentinel.py

```diff
@@ -1423,18 +1426,18 @@ def _check_scihub_response(response, test_json=True, query_string=None):
     except (requests.HTTPError, ValueError):
         msg = None
         try:
-            msg = response.headers["cause-message"]
-        except Exception:
+            msg = response.json()["error"]["message"]["value"]
+        except:
```

Comment:
Why you have removed ``Exception`` here (and in the following)?
Do you also want to catch e.g. ``KeyboardInterrupt`` and things like that?

Reply:
Oh, wow, I've always just assumed that plain `except` is equivalent to `except Exception` rather than `except BaseException` and was not aware that `KeyboardInterrupt` derives from `BaseException`. That's super useful to know. Thank you! I'll fix that for sure.